### PR TITLE
Make admin API to return template attributes in structured format

### PR DESCRIPTION
**Pull request by banterCZ**
A follow-up to #1033

### CODE REVIEW

File: enrollment-server/src/main/java/com/wultra/app/enrollmentserver/controller/api/admin/AdminController.java [hunk 1]
```diff
@@ -45,25 +49,40 @@ public class AdminController {
 
     private final OperationTemplateService operationTemplateService;
 
+    private final ObjectMapper objectMapper;
+
     @GetMapping("/template")
     public ObjectResponse<List<TemplateDetailResponse>> templates() {
```
Comment (petrdvorak) [inline enrollment-server/src/main/java/com/wultra/app/enrollmentserver/controller/api/admin/AdminController.java:55]: Not an ideal format of response - you cannot add any attributes to the response object. I recommend creating one more wrapper object and keeping the list under the `templates` key in it.

**Review by petrdvorak — CHANGES_REQUESTED** [review]
There is one recommendation in the response structure, but I did not get the returned format.

We need grouping by template name, then grouping by language, and then grouping by content type. I do not see it here.

**Comment by banterCZ** [comment]
> There is one recommendation in the response structure, but I did not get the returned format.
> 
> We need grouping by template name, then grouping by language, and then grouping by content type. I do not see it here.

I prepared grouping in PAC, not in ES

File: enrollment-server/src/main/java/com/wultra/app/enrollmentserver/controller/api/admin/AdminController.java (hunk 1 above)
Comment (zcgandcomp) [reply enrollment-server/src/main/java/com/wultra/app/enrollmentserver/controller/api/admin/AdminController.java:55]: I guess you mean (as used in others APIs)
API should return `ObjectResponse<TemplateListResponse>`
and create an object
`TemplateListResponse extends ArrayList<TemplateDetailResponse> `

Comment (banterCZ) [reply enrollment-server/src/main/java/com/wultra/app/enrollmentserver/controller/api/admin/AdminController.java:55]: Ok, I see.

**Review by petrdvorak — APPROVED** [review]
Looks OK

**Review by romanstrobl — APPROVED** [review]
Nice refactoring.

**Pull request merged**